### PR TITLE
Week 6 project - Weather App (Caroline Fahlgren and Julia Ros)

### PR DESCRIPTION
**Pull request by julialeonieros**
Link to my deployed project: https://app.netlify.com/sites/clever-murdock-6c222e/overview

My reflections on how this week's project turned out:
We are pretty happy with the result. We aimed at implementing the second design that was in the design folder. 

Things I'd like to have clarified or explained in more detail:
The app looks best on a mobile sized device. Also, the switch-statement (line 39) evaluates a different type of "weather data" than the weather displayed at the top ("weather.main" instead of "weather.description"), simply because ".main" has fever types of weather, which made it easier to evaluate. We would have thought that the logical operator `||` would be used in the switch, but somehow that doesn't work. `&&` however, works fine (any thought on that are appreciated :)

### CODE REVIEW

File: code/example.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "id": 1788,
+      "country": "SE",
+      "sunrise": 1613370215,
+      "sunset": 1613403625
+    },
+    "timezone": 3600,
+    "id": 2673730,
+    "name": "Stockholm",
+    "cod": 200
+  }
```
Comment (Pauan8) [inline code/example.json:43]: Remember to remove files that's not used in the project :)

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    const sunrise = new Date(data.sys.sunrise * 1000).toLocaleTimeString('sv-SE', {
+      hour: '2-digit',
+      minute: '2-digit',
+    })
+
+    const sunset = new Date(data.sys.sunset * 1000).toLocaleTimeString('sv-SE', {
+      hour: '2-digit',
+      minute: '2-digit',
+    })
```
Comment (Pauan8) [inline code/script.js:28]: These two variables look almost exactly the same, maybe you could have made a function for this instead?

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+const cityHeader = document.getElementById('city')
+const sunrise = document.getElementById('sunrise')
+const sunset = document.getElementById('sunset')
+
+fetch(apiUrlToday)
+  .then(response => {
+    if (response.ok) {
+      return response.json()
+    } else {
+      throw 'Oops, something went wrong!'
```
Comment (Pauan8) [inline code/script.js:15]: Good that you have a .catch on row 65 👍  But think about that if the response won't work the catch will be executed, so if/else/throw is not really needed here.

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    }
+  })
+  .then((data) => {
+    //prints weekday and temperature for the next five days 
+    const filteredForecast = data.list.filter(day => day.dt_txt.includes('12:00'))
+    filteredForecast.forEach(day => {
+      const forecastDay = new Date(day.dt_txt)
+      weeksWeather.innerHTML += `
+      <div class='weekdays-container'><div class='day-container'>${forecastDay.toLocaleString('en-us', {weekday: 'short'}).toLowerCase()}</div><div class='temp-container'>${day.main.temp.toFixed(0)} &#176</div></div>
```
Comment (Pauan8) [inline code/script.js:81]: Indentations and row formatting in the innerHTML would be nice for better readability.

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    }
+  })
+  .catch(error => console.error(error))
+
+fetch(apiUrlFiveDays)
+  .then(response => {
+    if (response.ok) {
+      return response.json()
+    } else {
+      throw 'Oops, something went wrong!'
```
Comment (Pauan8) [inline code/script.js:72]: Same as in the other fetch, if/else not needed

File: code/script.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
+const apiUrlToday = 'https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=3340cbe473b3001d4487c919d349bee2'
+const apiUrlFiveDays = 'https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=3340cbe473b3001d4487c919d349bee2'
+const todaysWeatherSmall = document.getElementById('todays-weather-small')
+const todaysWeather = document.getElementById('todays-weather')
+const weeksWeather = document.getElementById('weeks-weather')
+const cityHeader = document.getElementById('city')
+const sunrise = document.getElementById('sunrise')
+const sunset = document.getElementById('sunset')
+
```
Comment (Pauan8) [inline code/script.js:9]: No global variables? Well done! 💯

**Review by Pauan8 — APPROVED** [review]
Good structured code and a nice project! A+ 
Feel free to look at the few comments for future projects :)

**Review by KarolCislowski — APPROVED** [review]
This is a very nice and clean project 🌈 
Great job!

**Pull request closed without merging**